File: branches/Medusa/Source/Fantasy.Jobs/TaskOutput.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.XSerialization;
namespace Fantasy.Jobs
{
    [XSerializable("output", NamespaceUri=Consts.XNamespaceURI )]
    public class TaskOutput : IConditionalObject
    {
        [XAttribute("taskParameter")]
        public string TaskParameter { get; set; }

        [XAttribute("propertyName")]
        public string PropertyName { get; set; }

        [XAttribute("itemCategory")]
        public string ItemCategory { get; set; }

        [XAttribute("condition")]
        public string Condition { get; set; }
    }
}

File: Fantasy.Jobs/src/fantasy/jobs/TaskOutput.java
package fantasy.jobs;

import fantasy.xserialization.*;

@XSerializable(name = "output", namespaceUri=Consts.XNamespaceURI)
public class TaskOutput implements IConditionalObject
{

	@XAttribute(name = "taskParameter")
	private String privateTaskParameter;
	public final String getTaskParameter()
	{
		return privateTaskParameter;
	}
	public final void setTaskParameter(String value)
	{
		privateTaskParameter = value;
	}

	@XAttribute(name = "propertyName")
	private String privatePropertyName;
	public final String getPropertyName()
	{
		return privatePropertyName;
	}
	public final void setPropertyName(String value)
	{
		privatePropertyName = value;
	}

	@XAttribute(name = "itemCategory")
	private String privateItemCategory;
	public final String getItemCategory()
	{
		return privateItemCategory;
	}
	public final void setItemCategory(String value)
	{
		privateItemCategory = value;
	}

	@XAttribute(name = "condition")
	private String privateCondition;
	public final String getCondition()
	{
		return privateCondition;
	}
	public final void setCondition(String value)
	{
		privateCondition = value;
	}
}